src/app/components/Sorter/Sorter.jsx

```diff
@@ -61,31 +75,54 @@ class Sorter extends React.Component {
     this.setState({ active: false });
   }
 
-  handleOnClickOut() {
-    if (this.state.active) {
-      this.setState({ active: false });
-    }
+  /**
+   * renderResultsSort()
+   * The fuction that makes renders the sort options.
+   *
+   * @return {HTML Element}
+   */
+  renderResultsSort() {
+    return sortingOpts.map((d, i) => (
+      <option value={d.val} key={i} selected={ d.val === this.state.sortValue }>
```

Comment:
React will warn for using `selected` attribute. We don't have to have `selected` as React will take care of it but we need it for no-js case.